docs/manage/partitions/flex/flex-pricing-faq.md

```diff
@@ -75,4 +75,15 @@ The table below should give you a sense of how the number of partitions you use
 We recommend you configure partitions to have less than 5 TB per day flowing into them. This practice:
 
 * Reduces the cost of each query.
-* Results in faster queries, since less data has to be scanned.  
\ No newline at end of file
+* Results in faster queries, since less data has to be scanned.  
+
+## How to optimize the query using default scope?
+
+Default scope allows you to include or exclude the partitions in the search query, helping you to optimize the search cost. For example, consider if a query needs to run through 10 partitions which consumes about 10 GB of search data, narrowing the search query using the default scope can reduce the time and cost. You can achieve this by selecting the **Include this partition in default scope** checkbox when creating the partition. Let's say, out of 10 partitions you excluded two partitions, now, when you run the query without any `_index`, the search will only consider the included partitions, reducing the amount of data scanned and lowering the cost.
```

Comment:
selecting or deselecting

Reply:
Thanks for your feedback Krati, I have fixed your comments.